pkg/events/grpc/grpc.go

```diff
@@ -72,10 +73,16 @@ type marshaledEvent struct {
 // EventsServer streams events from a PubSub over gRPC.
 type EventsServer struct {
 	ctx    context.Context
+	pubsub events.PubSub
 	events events.Channel
 	filter events.IdentifierFilter
 }
 
+var (
+	evtStreamStart = events.Define("events.stream.start", "event stream start")
```

Comment:
Is there some guideline on wording here? @htdvisser @johanstokking
1. `event stream start`
2. `event stream started`
3. `start event stream`
4. `started event stream`
Which is the preferred version?
In NS I go for (2.), i.e. <subject> <action>

I think we should be consistent in these for better UX

Reply:
Important is that the event names are consistent (we won't change those). We can change the wording/translations to improve UX. Perhaps we can go through the list of event/error definitions sometime and make everything perfect, but right now I think we have more important things to do than this kind of goldplating.

Reply:
We had enough time to change wording of logging statements, so for sure we should have enough time to change the wording of user-facing event descriptions.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/issues/540